.github/workflows/ci.yml

```diff
@@ -147,15 +145,59 @@ jobs:
         python3 -m black --check --diff .
         python3 -m flake8 .
 
-    - name: Build
+    - name: Build Wheels
+      env:
+        SCCACHE_FILE: sccache-v0.2.15-x86_64-unknown-linux-musl
       run: |
-        cd ${GITHUB_WORKSPACE}/k8s
-
         # build graphscope wheels
-        sudo make graphscope-py3-package
+        mkdir -p ${HOME}/.m2 ${HOME}/.cache/sccache ${HOME}/.cargo/git ${HOME}/.cargo/registry
+        docker pull registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-vineyard:v0.3.11
+        docker run --rm \
+          -v ${GITHUB_WORKSPACE}:/work \
+          -v ${HOME}/.m2:/home/graphscope/.m2 \
+          -v ${HOME}/.cache/sccache:/home/graphscope/.cache/sccache \
+          -v ${HOME}/.cargo/git:/home/graphscope/.cargo/git \
+          -v ${HOME}/.cargo/registry:/home/graphscope/.cargo/registry \
+          registry.cn-hongkong.aliyuncs.com/graphscope/graphscope-vineyard:v0.3.11 \
```

Comment:
Move the mount options to the makefile. You could add an option to disable it, but please enable cache by default. You have have something like

```
NO_CACHE := false

ifeq ($(NO_CACHE), true)
    CACHE_MOUNT_OPTS =
else
    CACHE_MOUNT_OPTS = -v $.....
endif

graphscope-manylinux2014-py3:
    ....
    docker run --rm $CACHE_MOUNT_OPTS ....
```

Reply:
sccache could be installed to our graphscope-runtime image. As it is only used for building wheels.

Reply:
Let's see if this can work. Then I would move it to makefile.

Reply:
> 

+1